mgmt/RecordsConfig.cc

```diff
@@ -518,6 +518,8 @@ static const RecordElement RecordsConfig[] =
   ,
   {RECT_CONFIG, "proxy.config.http.parent_proxy.connect_attempts_timeout", RECD_INT, "30", RECU_DYNAMIC, RR_NULL, RECC_NULL, nullptr, RECA_NULL}
   ,
+  {RECT_CONFIG, "proxy.config.http.update_hostdb_on_parent_proxy_connect_errors", RECD_INT, "0", RECU_DYNAMIC, RR_NULL, RECC_INT, "[0-1]", RECA_NULL}
```

Comment:
This is a rather long name of a configuration ... :) And it doesn't feel properly "name spaced" ? How about

```
CONFIG proxy.config.hostdb.mark_parent_down INT 0
```

or perhaps

```
CONFIG proxy.config.http.parent_proxy.mark_down_hostdb INT 0
```